### DISCUSSION TITLE
Only nonportable version of app exists on scoop (Icaros)

### DISCUSSION THREAD

**Discussion by airworksman** [ScoopInstaller/Scoop#5451] [Help and Questions] [answered]
I'm trying to stick with only portable apps and I came across one that only exists as a non portable version on scoop even though a portable version does exist.

For Icaros I think the latest portable version is here: https://github.com/Xanashi/Icaros/releases/download/v3.3.0/Icaros_v3.3.0.zip

Is it just an oversight, or is there any particular reason the portable version cannot be added? Hope the portable one can be added, thanks.

**Comment by rashil2000** [comment, marked as answer]
The variant available in the Nonportable bucket is actually the portable one - see https://github.com/ScoopInstaller/Nonportable/blob/master/bucket/icaros-np.json

![image](https://user-images.githubusercontent.com/46838874/229343124-1ac1bd62-17cc-4885-8fe2-da0e669ccd73.png)

**Reply by airworksman** [reply]
Oh, good to know, but when I'm looking for a portable app it never actually occured to me to look in the nonportable bucket heh...

I guess the question now is, any reason it is in the nonportable bucket only?

**Reply by rashil2000** [reply]
The reason it's in the Nonportable bucket is because it uses the Register Server commands to register the DLLs that come in the .zip archive, and those commands require admin privileges.

**Reply by airworksman** [reply]
Appreciate the info, I guess next time I just need to look through the JSON file to see if it runs an installer or let me know if there is an easier way, thanks.

**Reply by rashil2000** [reply]
Run this: 
```
scoop config show_manifest True
```
this will show the manifest of apps before installing them.

**Reply by airworksman** [reply]
Perfect, thank you.
